package.json

```diff
@@ -32,17 +32,15 @@
     "babel-preset-stage-3": "^6.24.1",
     "eslint": "^4.9.0",
     "eslint-config-prettier": "^2.6.0",
-    "eslint-import-resolver-webpack": "^0.8.3",
-    "eslint-plugin-import": "^2.8.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-prettier": "^2.3.1",
     "eslint-plugin-react": "^7.4.0",
     "prettier": "^1.11.1"
   },
   "peerDependencies": {
     "@bpanel/bpanel-ui": "^0.0.7",
-    "@bpanel/bpanel-utils": "^0.0.7",
-    "bcoin": "git+https://github.com/bcoin-org/bcoin.git#wallet-rewrite",
+    "@bpanel/bpanel-utils": "*",
```

Comment:
This does depend on at least a certain version of `bpanel-utils` because the `getClient` architecture wasn't always there

Reply:
Yeah, that's true. Will update.